src/main/kotlin/com/playwright/remote/engine/route/UrlMatcher.kt

```diff
@@ -26,6 +26,8 @@ class UrlMatcher(private val rawSource: Any = "", private val predicate: Predica
         }
     }
 
+    constructor() : this("", null)
```

Comment:
If you will set default value to null for predicate, this default constructor should be just
`constructor() : this("")` as per previous comment.

Reply:
Fixed